spark/dl/pom.xml

```diff
@@ -252,17 +251,6 @@
                     </execution>
                 </executions>
             </plugin>
-
-            <plugin>
-                <groupId>net.alchim31.maven</groupId>
-                <artifactId>scala-maven-plugin</artifactId>
```

Comment:
Why delete scala-maven-plugin?

The scala-maven-plugin is used for compiling/testing/running/documenting scala code in maven.